src/objects/Hierarchy_Settings__c.object

```diff
@@ -77,6 +77,28 @@
         <type>Checkbox</type>
     </fields>
     <fields>
+        <fullName>Default_Enrolled_Student_RecordType_Id__c</fullName>
```

Comment:
I renamed them to match the current format. They are Student_RecType__c and Faculty_RecType__c.